### PR TITLE
[Fluent] Add expansion-panel styles

### PR DESCRIPTION
**Pull request by kikostadinov**
Related to: https://github.com/telerik/kendo-themes-private/issues/3

### CODE REVIEW

File: packages/fluent/scss/expansion-panel/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+@use "_variables.scss" as *;
+@use "../core/color-system" as *;
+
+@mixin kendo-expansion-panel--layout() {
+
+    // Base
+    .k-expander {
+        border-width: $kendo-expander-border-width;
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/_layout.scss:8]: all variables should be replaced with CSS vars

File: packages/fluent/scss/expansion-panel/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    // Header
+    .k-expander-header {
+        padding: $kendo-expander-header-padding-y $kendo-expander-header-padding-x;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        cursor: pointer;
+        margin-top: $kendo-expander-header-margin-y;
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/_layout.scss:35]: use margin-block-start

File: packages/fluent/scss/expansion-panel/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    .k-expander {
+        @include fill(
+            var( --kendo-expander-text, #{$kendo-expander-text} ),
+            var( --kendo-expander-bg, #{$kendo-expander-bg} ),
+            var( --kendo-expander-border, #{$kendo-expander-border} ),
+        );
+        @include box-shadow( var( --kendo-expander-box-shadow, #{$kendo-expander-box-shadow} ) );
+
+        &.k-state-focus,
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/_theme.scss:15]: remove -state classes - k-state-focus, k-state-hover, etc

File: packages/fluent/scss/expansion-panel/_theme.scss [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        &.k-focus {
+            @include box-shadow( var( --kendo-expander-focus-shadow, #{$kendo-expander-focus-shadow} ) );
+        }
+
+        &.k-expanded {
+            .k-expander-header {
+                background-color: $kendo-expander-expanded-header-bg;
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/_theme.scss:22]: use CSS var

File: packages/fluent/scss/expansion-panel/_variables.scss [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+/// Horizontal margin of the expander indicator.
+/// @group expander
+$kendo-expander-indicator-margin-x: map-get( $kendo-spacing, 2 ) + map-get( $kendo-spacing, thin ) !default;
+/// Text color of the expander indicator.
+/// @group expander
+$kendo-expander-indicator-text: $kendo-expander-text;
+
+/// Horizontal padding of the expander content.
+/// @group expander
+$kendo-expander-content-padding-x: $kendo-padding-x * 2 !default;
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/_variables.scss:88]: use calc

File: packages/fluent/scss/expansion-panel/index.scss [hunk 6]
```diff
@@ -0,0 +1,22 @@
+// Module meta
+$_kendo-module-meta: (
+    name: "expansion-panel",
+    dependencies: ()
```
Comment (TeyaVes) [inline packages/fluent/scss/expansion-panel/index.scss:4]: add icon as a dependency

**Review by TeyaVes — CHANGES_REQUESTED** [review]

File: packages/fluent/scss/expansion-panel/_layout.scss [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+        display: flex;
+        flex-direction: column;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: $rgba-transparent;
+
+        &:not(:last-child) {
+            &:not(.k-focus) {
+                margin-block-end: var( --kendo-expander-margin-y, #{$kendo-expander-margin-y} );
+            }
+        }
```
Comment (Juveniel) [inline packages/fluent/scss/expansion-panel/_layout.scss:25]: why do we need this margin ?

File: tests/expansion-panel/expansion-panel.html [hunk 8]
```diff
@@ -16,7 +16,7 @@
                 <div class="k-expander-sub-title">South America</div>
                 <span class="k-expander-indicator k-icon k-i-arrow-chevron-down"></span>
             </div>
-            <div class="k-expander-content-wrapper k-display-none">
+            <div class="k-expander-content-wrapper k-d-none">
```
Comment (Juveniel) [inline tests/expansion-panel/expansion-panel.html]: You should not update markup in the /tests folder.  These are snapshots generated from the html components that are available in the `/packages/html` folder. In order to change the actual tests, please do so by modifying the real test in `/packages/html/src/expansion-panel/tests`.

Revert this change afterwards.

**Review by Juveniel — CHANGES_REQUESTED** [review]

File: packages/fluent/scss/expansion-panel/_layout.scss (hunk 7 above)
Comment (kikostadinov) [reply packages/fluent/scss/expansion-panel/_layout.scss]: I was looking for a way to remove one of the horizontal shadows when the panels are in a group. There is a different solution in the next commit.

**Review by TeyaVes — APPROVED** [review]

**Review by Juveniel — APPROVED** [review]
